contracts/Relayer.sol

```diff
@@ -44,6 +46,11 @@ contract Relayer is IRelayer {
         _;
     }
 
+    modifier _onlyBridge() {
```

Comment:
`_bridgeAddress` is never set anywhere, so this modifier will always fail. I made a PR with my proposed changes: #100